src/commands/wait_for_job.yml

```diff
@@ -27,6 +27,12 @@ parameters:
       '*' the wait will run only on the specified branch
     type: string
     default: "*"
+  circleci-api-key:
```

Comment:
interesting, we should do this in all commands. note to self

Reply:
Yeah, that would make the orb a little more flexible!